web3-dev/smart-contracts/playground.rst

```diff
@@ -0,0 +1,14 @@
+.. _sc-playground:
+
+Smart-contract Playground
+=========================
+
+`Smart-contract Playground <https://massa.net/sc-playground>`_ is a simple interface for writing, unit-testing, compiling, and sharing the smart-contract code written in Assembly Script. 
+
+Smart-contract Playground runs in your browser's local storage.
+
+At the moment, Playground is limited in terms of imported smart-contract methods.
+The available methods are:
```

Comment:
Methods are not formatted well. See https://massa--2.org.readthedocs.build/en/2/web3-dev/smart-contracts/playground.html

Reply:
I've just pushed the fix

Reply:
![image](https://user-images.githubusercontent.com/11590067/199059397-0071afe5-c35b-43ee-85ee-928e08945225.png)